main/src/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1952,4 +1952,27 @@ public static String getUsername(final String page) {
         return StringUtils.isNotEmpty(usernameOld) ? usernameOld : null;
     }
 
+    @Nullable
```

Comment:
@Nullable/@NonNull is for objects

really `cachesCount=-1` if it could not determined?
@Lineflyer What do you mean from support point of view?

Reply:
Copy/paste mistake